bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gaia-component",
-  "version": "0.3.6",
+  "version": "0.3.7",
```

Comment:
Remember to push a matching git tag when this lands. Usually I would stamp versions in a separate commit then tag that commit, then push straight to `master`. But this doesn't matter that much :)